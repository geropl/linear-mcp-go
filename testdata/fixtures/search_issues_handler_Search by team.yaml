---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 703
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tquery SearchIssues($filter: IssueFilter, $first: Int, $includeArchived: Boolean) {\n\t\t\tissues(filter: $filter, first: $first, includeArchived: $includeArchived) {\n\t\t\t\tnodes {\n\t\t\t\t\tid\n\t\t\t\t\tidentifier\n\t\t\t\t\ttitle\n\t\t\t\t\tdescription\n\t\t\t\t\tpriority\n\t\t\t\t\turl\n\t\t\t\t\tstate {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tname\n\t\t\t\t\t}\n\t\t\t\t\tassignee {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tname\n\t\t\t\t\t}\n\t\t\t\t\tlabels {\n\t\t\t\t\t\tnodes {\n\t\t\t\t\t\t\tid\n\t\t\t\t\t\t\tname\n\t\t\t\t\t\t}\n\t\t\t\t\t}\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"filter":{"team":{"id":{"eq":"234c5451-a839-4c8f-98d9-da00973f1060"}}},"first":5,"includeArchived":false}}'
        form: {}
        headers:
            Content-Type:
                - application/json
            User-Agent:
                - linear-mcp-go/1.0.0
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: "{\"data\":{\"issues\":{\"nodes\":[{\"id\":\"9407c793-5fd8-4730-9280-0e17ffddf320\",\"identifier\":\"TEST-12\",\"title\":\"Comments issue\",\"description\":\"This is the description\",\"priority\":0,\"url\":\"https://linear.app/linear-mcp-go-test/issue/TEST-12/comments-issue\",\"state\":{\"id\":\"cffb8999-f10e-447d-9672-8faf5b06ac67\",\"name\":\"Todo\"},\"assignee\":null,\"labels\":{\"nodes\":[]}},{\"id\":\"f284f228-2aa7-42ad-82b3-fed96c6df3b7\",\"identifier\":\"TEST-11\",\"title\":\"Test Issue\",\"description\":null,\"priority\":0,\"url\":\"https://linear.app/linear-mcp-go-test/issue/TEST-11/test-issue\",\"state\":{\"id\":\"42f7ad15-fca3-4d11-b349-0e3c1385c256\",\"name\":\"Backlog\"},\"assignee\":null,\"labels\":{\"nodes\":[]}},{\"id\":\"1c2de93f-4321-4015-bfde-ee893ef7976f\",\"identifier\":\"TEST-10\",\"title\":\"Updated Test Issue\",\"description\":null,\"priority\":0,\"url\":\"https://linear.app/linear-mcp-go-test/issue/TEST-10/updated-test-issue\",\"state\":{\"id\":\"42f7ad15-fca3-4d11-b349-0e3c1385c256\",\"name\":\"Backlog\"},\"assignee\":null,\"labels\":{\"nodes\":[]}},{\"id\":\"dec18d37-c4a1-4fac-bcea-2a37de58d607\",\"identifier\":\"TEST-9\",\"title\":\"Next steps\",\"description\":\"You're all set to use Linear. \\n\\n**Want to learn more?**\\n\\n* Check out our [docs](https://linear.app/docs) to dive into specific features\\n* Learn about new features in weekly[ changelogs](https://linear.app/changelog)\\n* Read the [Linear Method](https://linear.app/linear-method), our approach to software building\\n* Join our Linear Customer Slack at this [link](http://linear.app/join-slack)\\n\\n**ProTip: How to trial Linear on your team**\\n\\n* Use our [in-app importer](https://linear.app/settings/import-export) to add issues from Jira, Asana, Shortcut or GitHub\\n* Trial it on your team. Pick a project or sprint, invite a few coworkers, add key integrations like Slack and GitHub to experience the full workflow.\\n\\n## Have a question?\\n\\nReach out to us via the *Contact us* option by hitting `?`.\",\"priority\":4,\"url\":\"https://linear.app/linear-mcp-go-test/issue/TEST-9/next-steps\",\"state\":{\"id\":\"cffb8999-f10e-447d-9672-8faf5b06ac67\",\"name\":\"Todo\"},\"assignee\":null,\"labels\":{\"nodes\":[]}},{\"id\":\"d510cb89-e29d-478f-884b-83cf59c079ab\",\"identifier\":\"TEST-3\",\"title\":\"Connect to Slack\",\"description\":\"If your team uses Slack, enable this integration to keep your workflow and communication in sync. \\n\\n* See issue updates in a dedicated channel.\\n* Use the `/linear` command to create new issues from Slack messages.\\n* Sync comments between Slack and Linear.\\n* Set up personal Slack notifications.\\n* Import custom emoji \U0001F973 to your Linear workspace.\\n\\n[Connect Slack â†’](https://linear.app/settings/integrations/slack)\",\"priority\":3,\"url\":\"https://linear.app/linear-mcp-go-test/issue/TEST-3/connect-to-slack\",\"state\":{\"id\":\"cffb8999-f10e-447d-9672-8faf5b06ac67\",\"name\":\"Todo\"},\"assignee\":null,\"labels\":{\"nodes\":[]}}]}}}\n"
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"afd-zwPEhTTIPXgxSCHh7C8sI6gJaHE"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 81.081645ms
