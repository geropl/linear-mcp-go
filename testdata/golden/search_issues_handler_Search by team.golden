{
  "err": "",
  "output": "Found 5 issues:\n- TEST-12: Comments issue\n  Priority: None\n  Status: Todo\n  https://linear.app/linear-mcp-go-test/issue/TEST-12/comments-issue\n- TEST-11: Test Issue\n  Priority: None\n  Status: Backlog\n  https://linear.app/linear-mcp-go-test/issue/TEST-11/test-issue\n- TEST-10: Updated Test Issue\n  Priority: None\n  Status: Backlog\n  https://linear.app/linear-mcp-go-test/issue/TEST-10/updated-test-issue\n- TEST-9: Next steps\n  Priority: 4\n  Status: Todo\n  https://linear.app/linear-mcp-go-test/issue/TEST-9/next-steps\n- TEST-3: Connect to Slack\n  Priority: 3\n  Status: Todo\n  https://linear.app/linear-mcp-go-test/issue/TEST-3/connect-to-slack\n"
}